nvflare/__init__.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "2.0.5"
+__version__ = "2.0.7"
```

Comment:
should this be 2.0.8?

Reply:
When we have CI properly set up, this can be the actual release version.  For now, I have to change it locally before building the wheel package.  So the repo might be one release lag.